core/vm/jump_table.go

```diff
@@ -1005,6 +1005,12 @@ func newFrontierInstructionSet() JumpTable {
 			maxStack:   maxStack(2, 0),
 			memorySize: memoryReturn,
 		},
+		INVALID: {
+			execute:     opInvalid,
+			constantGas: 0,
+			minStack:    minStack(0, 0),
+			maxStack:    maxStack(0, 0),
+		},
```

Comment:
Though -- in the context of what you're implementing in that other PR... Is it really necessary to define it as an acutal op in the jumptable, as opposed to just define the `OpCode` but leave it empty in the jumptable?

Reply:
I think it's better to have it in the jumptable, I see the question of whether 0xfe is a defined opcode as a property of particular fork rules (currently it's defined in all forks, but theoretically 0xfe might mean something else in some other jump table)

Without putting it into jumptable I can only think of having it as a special case in code validation, like `jumpTable[opcode] == nil && opcode != 0xfe` will mean it's not defined.